Makefile

```diff
@@ -27,11 +34,23 @@ config:: # Configure development environment (main) @Configuration
 	make \
 		terraform-install
 
-# ==============================================================================
-
-${VERBOSE}.SILENT: \
```

Comment:
Can we keep `${VERBOSE}.SILENT`, please? The desired behaviour is that when the `VERBOSE` flag is set, make targets are no longer silent.